examples/subscribe.rs

```diff
@@ -40,36 +38,25 @@ async fn main() -> Result<(), Box<dyn snafu::Error>> {
 
     println!("running!");
 
-    client
-        .set_presence_state(HashMap::<String, String>::from([
-            (
-                "is_doing".to_string(),
-                "Nothing... Just hanging around...".to_string(),
-            ),
-            ("flag".to_string(), "false".to_string()),
-        ]))
-        .channels(["my_channel".into(), "other_channel".into()].to_vec())
-        .user_id("user_id")
-        .execute()
-        .await?;
-
     tokio::time::sleep(tokio::time::Duration::from_secs(3)).await;
 
-    let subscription = client.subscription(SubscriptionParams {
+    let subscription = pubnub.subscription(SubscriptionParams {
         channels: Some(&["my_channel", "other_channel"]),
         channel_groups: None,
         options: Some(vec![SubscriptionOptions::ReceivePresenceEvents]),
     });
-    subscription.subscribe(None);
+    subscription.subscribe();
```

Comment:
I love that getting rid of this meaningless `None` in `subscribe()`. 

But i'm still thinking about changing the `SubscriptionParams` into builder. What do you think?

Reply:
Maybe in next changes because @techwritermat almost done with docs.